.github/release-please.yml

```diff
@@ -0,0 +1,9 @@
+handleGHRelease: true
```

Comment:
This will use the app now right? If so, could you remove the two release please actions.